cms/core/utils.py

```diff
@@ -30,13 +29,13 @@ def get_formatted_pages_list(
                 "url": page.get_url(request=request),
             },
             "metadata": {
-                "object": {"text": getattr(page, "label", _("Page"))},
+                "object": {"text": getattr(page, "label", "Page")},
             },
             "description": getattr(page, "listing_summary", "") or getattr(page, "summary", ""),
         }
         if release_date := page.release_date:
             datum["metadata"]["date"] = {
-                "prefix": _("Released"),
+                "prefix": "Released",
```

Comment:
these two need to be translatable as they show on the FE. Topic page, for example

Reply:
Addressed in [20c4f5d](https://github.com/ONSdigital/dis-wagtail/pull/136/commits/20c4f5daa1e25748ca92bf084aa758b6c855f761)